mobile-save-for-later/template.yml

```diff
@@ -0,0 +1,97 @@
+# This AWS SAM template has been generated from your function's configuration. If
```

Comment:
I think CDK is used to define the cloudformation stack for this service?  Is it used for another purpose?

Reply:
Yeah, we are using the CDK for the cfn stack definition. This is used to be able to run the lambda functions locally using aws sam. Instructions here: https://github.com/guardian/mobile-save-for-later/pull/102/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R86